src/main/java/uk/gov/hmcts/reform/pip/data/management/database/ArtefactRepository.java

```diff
@@ -75,53 +75,58 @@ List<Artefact> findArtefactBySearch(@Param(SEARCH_TERM_PARAM) String searchTerm,
 
     @Query(value = "select location_id, count(distinct artefact_id) from artefact "
         + "where location_id ~ '^[0-9]+$' "
-        + "and is_archived != true "
+        + "and (display_to > :curr_date or display_to is null)"
         + "group by location_id",
         nativeQuery = true)
-    List<Object[]> countArtefactsByLocation();
+    List<Object[]> countArtefactsByLocation(@Param(CURRENT_DATE_PARAM) LocalDateTime currentDate);
 
-    @Query(value = "select * from Artefact where location_id = :location_id and is_archived != true",
+    @Query(value = "select * from Artefact "
+        + "where location_id = :location_id "
+        + "and (display_to > :curr_date or display_to is null)",
         nativeQuery = true)
-    List<Artefact> findArtefactsByLocationIdAdmin(@Param(LOCATION_ID_PARAM) String locationId);
+    List<Artefact> findArtefactsByLocationIdAdmin(@Param(LOCATION_ID_PARAM) String locationId,
+                                                  @Param(CURRENT_DATE_PARAM) LocalDateTime currentDate);
 
     @Query(value = "select * from Artefact where artefact_id = CAST(:artefact_id AS uuid)",
         nativeQuery = true)
     Optional<Artefact> findArtefactByArtefactId(@Param(ARTEFACT_ID_PARAM) String artefactId);
 
-    @Query(value = "SELECT * FROM Artefact WHERE DATE(display_from) = :curr_date and is_archived != true",
+    @Query(value = "SELECT * FROM Artefact "
+        + "WHERE DATE(display_from) = :curr_date "
+        + "AND (display_to > :curr_date or display_to is null)",
         nativeQuery = true)
-    List<Artefact> findArtefactsByDisplayFrom(@Param(CURRENT_DATE_PARAM) LocalDate today);
+    List<Artefact> findArtefactsByDisplayFrom(@Param(CURRENT_DATE_PARAM) LocalDate today,
+                                              @Param(CURRENT_DATE_PARAM) LocalDateTime currentDate);
```

Comment:
CURRENT_DATE_PARAM is used for both today and currentDate, you only needs one

Reply:
as above, display to has time in it. but you comment was correct. I was only checking date. I made the fix.